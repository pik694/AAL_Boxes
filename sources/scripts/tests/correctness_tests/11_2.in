11
81 29 53
65 99 62
22 92 3
1 62 13
25 10 35
45 50 58
1 6 97
10 40 95
95 68 81
55 6 70
37 31 92

