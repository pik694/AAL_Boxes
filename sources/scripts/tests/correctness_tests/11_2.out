6 3 2 7 4 10 9 0 1 5 8 
1360061->1089436
